tools/install.sh

```diff
@@ -397,14 +423,18 @@ if [ -z "$KING_PHISHER_SKIP_SERVER" ]; then
 	elif [ "$LINUX_VERSION" == "Ubuntu" ]; then
 		echo "Installing the King Phisher upstart service file in /etc/init/"
 		cp data/server/service_files/king-phisher.conf /etc/init
-		sed -i -re "s|^exec KingPhisherServer|# exec KingPhisherServer|" /etc/init/king-phisher.conf
-		sed -i -re "s|^#\\s?exec /opt|exec /opt|" /etc/init/king-phisher.conf
 		sed -i -re "s|/opt\/king-phisher|$KING_PHISHER_DIR|g" /etc/init/king-phisher.conf
 
 		echo "Starting the King Phisher service"
 		start king-phisher
 	else
-		echo "Start the King Phisher server with the following command: "
-		echo "sudo $KING_PHISHER_DIR/KingPhisherServer -L INFO -f $KING_PHISHER_DIR/server_config.yml"
+		echo "-----------------------------------------------------------------------------------------------"
+		echo "Start the King Phisher server with the following command prior to starting the client: "
+		echo "sudo python3 $KING_PHISHER_DIR/KingPhisherServer -L INFO -f $KING_PHISHER_DIR/server_config.yml"
 	fi
 fi
+if [ -z "$KING_PHISHER_SKIP_CLIENT" ]; then
+	echo "-----------------------------------------------------------------------------------------------"
+	echo "You can start King Phisher client with the following command: "
```

Comment:
`You can start the King Phisher client with the following command:` (Added the "the" word and dropped the trailing space.)